examples/3d/elixir_euler_eoc_test.jl

```diff
@@ -0,0 +1,60 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(2.0)
+
+initial_condition = Trixi.initial_condition_eoc_test_coupled_euler_gravity
+
+surface_flux = flux_hll
+volume_integral = VolumeIntegralWeakForm()
+solver = DGSEM(3, surface_flux, volume_integral)
+
+coordinates_min = (0, 0, 0)
+coordinates_max = (2, 2, 2)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=2,
+                n_cells_max=10_000)
+
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver,
+                                    source_terms=Trixi.source_terms_eoc_test_euler)
```

Comment:
This one should also be exported (see comment above).

Reply:
It wasn't, but is in 2f7f549.